browser/BUILD.gn

```diff
@@ -425,6 +425,7 @@ source_set("browser_process") {
 
 group("browser_process_public") {
   public_deps = [
+    "//brave/chromium_src/chrome/app:constants",
```

Comment:
this is going to change from @darkdh's PR, but I suppose we could merge and then update for both

Reply:
https://github.com/brave/brave-browser/issues/12562